packages/gatsby-theme-api-docs/index.js

```diff
@@ -1,4 +1,9 @@
-// Export only components that must be injected into the MDX provider
-export { default as ApiType } from './src/components/type';
+export { default as ApiType } from './src/components/type/type-by-key';
 export { default as ApiEndpoint } from './src/components/resource-method';
-export { default as ApiEndpointsForResource } from './src/components/resource/resource';
+export { default as ApiEndpointsForResource } from './src/components/resource/resource-by-key';
+export * from './src/hooks/use-api-types';
+export * from './src/hooks/use-api-resources';
+export * from './src/hooks/use-apis';
+export * from './src/hooks/use-type-location-overrides';
+export * from './src/hooks/use-type-locations';
+export * from './src/hooks/use-read-resource-by-resource-path';
```

Comment:
What's the reason for exporting those?

Reply:
The intent is to be enable building site specific API components that use the same data.  Every additional hook that loads nearly-the-same-but-different data just adds to the page weight.   This set of hooks is a client-side data interface to the API specs, exporting it from the package root communicates that you should always use these instead of adding own additional static queries on the same data. 

There is no specific use case I want to build right now.   So removing them would not hurt right now.

Reply:
I see. Ok then.